Makefile

```diff
@@ -47,7 +47,7 @@ build: dist/ec_$(shell go env GOOS)_$(shell go env GOARCH) ## Build the ec binar
 
 .PHONY: test
 test: ## Run unit tests
-	@go test -race -covermode=atomic -coverprofile=coverage-unit.out -short -timeout 500ms ./...
+	@go test -race -covermode=atomic -coverprofile=coverage-unit.out -short -timeout 30s ./...
```

Comment:
Can we add a comment here explaining the 30s value? The PR description does a good job at explaining and also mentions that this should be reverted back to 500ms at some point. That will not be obvious to someone looking at this file.

Reply:
Updated in #108